experimental/packages/exporter-trace-otlp-http/test/traceHelper.ts

```diff
@@ -397,26 +404,20 @@ export function ensureExportTraceServiceRequestIsSet(
     'resourceSpans is missing'
   );
 
-  const resource = resourceSpans[0].resource;
+  const resource = resourceSpans?.[0].resource;
   assert.strictEqual(!!resource, true, 'resource is missing');
 
-  const instrumentationLibrarySpans =
-    resourceSpans[0].instrumentationLibrarySpans;
+  const scopeSpans = resourceSpans?.[0].scopeSpans;
   assert.strictEqual(
-    instrumentationLibrarySpans && instrumentationLibrarySpans.length,
+    scopeSpans && scopeSpans.length,
```

Comment:
nit: can also be shortened to `scopeSpan?.length` I think

Reply:
Yep, that's better. Thanks! :slightly_smiling_face:  Cleaned up several occurences of this in [c33fbe1](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/c33fbe1c321c8e118894a7ccecd164baffab20be)

Reply:
Also possible to do `assert.ok(scopeSpans)` above then the `?` is not needed (will also fail the tests if scopeSpans is missing) but the way you did is also fine.